Lib/dataclasses.py

```diff
@@ -212,6 +208,9 @@ def __init__(self, type):
     def __repr__(self):
         return f'dataclasses.InitVar[{self.type.__name__}]'
 
+    def __class_getitem__(cls, params):
```

Comment:
These changes aren't related to the title of this PR.   Please either split it out into a separate PR, or retitle this PR.

Reply:
Oh, yes. They are from other issue.